apps/website/src/examples/Alert.sticky.tsx

```diff
@@ -8,15 +8,26 @@ import { Alert } from '@itwin/itwinui-react';
 export default () => {
   return (
     <div>
-      <Alert
-        isSticky
-        onClose={() => console.log('CLOSED')}
-        clickableTextProps={{ onClick: () => console.log('Clicked more info!') }}
-        clickableText='Learn more'
-      >
-        This is a sticky alert
+      <Alert isSticky>
+        <Alert.Icon />
+        <Alert.Message>
+          This is a sticky alert
+          <Alert.Action onClick={() => console.log('Clicked more info!')}>Learn more</Alert.Action>
+        </Alert.Message>
+        <Alert.CloseButton onClick={() => console.log('CLOSED')} />
       </Alert>
       <p>Page content.</p>
     </div>
   );
 };
+
+{
+  /* <Alert
+isSticky
+onClose={() => console.log('CLOSED')}
+clickableTextProps={{ onClick: () => console.log('Clicked more info!') }}
+clickableText='Learn more'
+>
+This is a sticky alert
+</Alert> */
+}
```

Comment:
Commented code should be removed before merging.

Reply:
Removed